js-ext-core/src/ethers-v6/unit.ts

```diff
@@ -0,0 +1,42 @@
+import { BigNumberish } from "@ethersproject/bignumber";
+
+import {
+  parseKaia as parseKaiaBase,
+  parseKaiaUnits as parseKaiaUnitsBase,
+  parseUnits as parseUnitsBase,
+} from "../util";
+/**
+ * Convert kaia to kei
+ *
+ * @param   kaia  string number in kaia unit
+ * @returns equivalent value in kei.
+ */
+export function parseKaia(kaia: string): bigint {
+  return parseKaiaBase(kaia).toBigInt();
```

Comment:
@Sotatek-TanHoang Is the backward compatibility broken?

Reply:
@kjeom Do you mean can user use this v6 function for v5?